tests/sentry/utils/appleconnect/test_appstore_connect.py

```diff
@@ -6,17 +8,18 @@
 
 class TestGetDsymUrl:
     def test_none_bundles(self) -> None:
-        assert appstore_connect._get_dsym_url(None) is NoDsymUrl.NOT_NEEDED
+        assert appstore_connect._get_dsym_url(None) == NoDsymUrl.NOT_NEEDED
```

Comment:
these comparisons were correct with `is` afaik, why did you change them?
https://docs.python.org/3/library/enum.html?#comparisons